client/my-sites/domains/domain-management/edit/registered-domain.jsx

```diff
@@ -189,14 +189,16 @@ const RegisteredDomain = createReactClass( {
 	},
 
 	contactsPrivacyNavItem() {
+		const { privacyAvailable } = this.props.domain;
```

Comment:
You can do nested destructuring:
```const { initialContext: { query }, step } = this.props;```

Tho separate ones read better, I guess. :)